sdk/eventhub/azure-eventhubs/azure/eventhub/_consumer_producer_mixin.py

```diff
@@ -7,12 +7,31 @@
 import logging
 import time
 
-from uamqp import errors
+from uamqp import errors, constants, compat
```

Comment:
Nitpick: from a code style perspective, I generally find it better to import the module and refer to the module qualified name to distinguish between external things and things defined in my module. I find that it helps with readability. @bryevdv , @lmazuel , opinions?

Reply:
In general I tend to prefer to import names from modules, e.g `from foo import a, b, c` but I feel like this comment is actually about something else and I am missing some context

Reply:
I'm like @bryevdv , I tend to prefer `from foo import a, b, c`. If I remember correctly, @brettcannon tends to prefer `from foo import bar; bar.a`
For some reason though, maybe because the strings are really generic (errors, constants, etc.), I feel `uamqp.errors` would be better.

Reply:
thank for all the discussion.

I would take this as improving readability of code related to generic imported modules. 
**e.g.** in _consumer_producer_mixin.py, both **uamqp error** and **eventhub error** exist. So it would be better to add the module name when using the error.

This could be done as an improvement in the next preview.

Reply:
My reasoning for preferring importing to the module instead of to objects in modules is two-fold. One is what @johanste said; I find it much more readable to know when an object originating outside the module without having to scroll to the top to look at the imports.

The second is technical. If you import objects instead of modules you are running the risk of having a circular import. Imports to the module, though, have a much lower chance due to how import works.